src/system/NewDialog/NewDialog.js

```diff
@@ -56,7 +56,7 @@ export const NewDialog = ( {
 					<DialogTitle title={ title } hidden={ ! showHeading } />
 					<DialogDescription description={ description } hidden={ ! showHeading } />
 
-					{ content }
+					<div role="document">{ content }</div>
```

Comment:
Out of curiosity, why don't we use role=dialog? 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role

Reply:
We already have that in the dialog. This is just for the content.

Reply:
oh, I see